src/common/IsolatedWorld.ts

```diff
@@ -117,7 +117,7 @@ export class IsolatedWorld {
   #frame: Frame;
   #injected: boolean;
   #document?: ElementHandle<Document>;
-  #contextPromise = createDeferredPromise<ExecutionContext>();
+  #context = createDeferredPromise<ExecutionContext>();
```

Comment:
revert the rename as it's not the context itself but a promise for a context?

Reply:
It's better to just keep it as `context` or else we would have to litter the space with `Promise`. We shouldn't care if it's a promise unless we *need* to make the distinction.